tensorflow/buildspec.yml

```diff
@@ -44,6 +44,28 @@
       docker_file: !join [ docker/, *VERSION, /, *DOCKER_PYTHON_VERSION, /, *CUDA_VERSION, /Dockerfile., *DEVICE_TYPE ]
       context:
         <<: *TRAINING_CONTEXT
```

Comment:
Can you add these lines in context section-
`    inference_context: &INFERENCE_CONTEXT
      sagemaker_package_name:
        source: docker/build_artifacts/sagemaker
        target: sagemaker
      deep_learning_container:
        source: docker/build_artifacts/deep_learning_container.py
        target: deep_learning_container.py
      init:
        source: docker/build_artifacts/__init__.py
        target: __init__.py
      dockerd-entrypoint:
        source: docker/build_artifacts/dockerd-entrypoint.py
        target: dockerd-entrypoint.py`

And these lines in repo_info-
`    inference_repository: &INFERENCE_REPOSITORY
      image_type: &INFERENCE_IMAGE_TYPE inference
      root: !join [ *FRAMEWORK, "/", *INFERENCE_IMAGE_TYPE ]
      repository_name: &REPOSITORY_NAME !join [pr, "-", *FRAMEWORK, "-", *INFERENCE_IMAGE_TYPE]
      repository: &REPOSITORY !join [ *ACCOUNT_ID, .dkr.ecr., *REGION, .amazonaws.com/, *REPOSITORY_NAME ]`

Reply:
Done.